<<<<<<< HEAD
<?xml version="1.0" encoding="UTF-8"?>
=======
>>>>>>> 62ffced7a03b4bcf911f1b212369ade8808337d3
<ui version="4.0">
 <class>ExportDialog</class>
 <widget class="QDialog" name="ExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
<<<<<<< HEAD
    <width>693</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPathPatient">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>C:/</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPathFile">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>671</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>C:/</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lbListItems">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>551</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Recordings of patient</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Export file:</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>801</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Patient file:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>360</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExport">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>360</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>export files</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSelectExportFiles">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Select export file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSelectPatient">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>select patient</string>
   </property>
  </widget>
=======
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
>>>>>>> 62ffced7a03b4bcf911f1b212369ade8808337d3
 </widget>
 <resources/>
 <connections/>
</ui>
